Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:22.13.1-alpine
+LABEL author="internxt"
+
+WORKDIR /usr/app
+
+COPY package.json ./
+COPY yarn.lock ./
+
+# Copy .npmrc file if it exists
+RUN [ -f .npmrc ] && cp .npmrc /usr/app/ || echo "npmrc file not found"
```

Comment:
The build and deploy action do the job regarding to this one, this line is not needed.